spec/features/spending_all_kudos_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+# frozen_string_literal: true
+RSpec.describe 'Kudo spec', type: :feature do
+  let!(:kudo) { build(:kudo) }
+
+  it 'spending all kudos' do
+    visit root_path
+
+    fill_in 'employee[email]', with: kudo.giver.email
+    fill_in 'employee[password]', with: kudo.giver.password
+    click_button 'Log in'
+
+    expect(page).to have_content 'success'
+
+    i = 10
```

Comment:
I would suggest to change hardcoded data in tests, and put there data from our testing objects.
```suggestion
    i = kudo.giver.number_of_available_kudos
```
also, we can remove `i` from this test and use everywhere `kudo.giver.number_of_available_kudos`